store/postgres/src/dynds/private.rs

```diff
@@ -27,20 +52,49 @@ impl DataSourcesTable {
             create table {nsp}.{table} (
                 vid integer generated by default as identity primary key,
                 block_range int4range not null,
-                access_group integer generated by default as identity,
+                causality_region integer generated by default as identity,
```

Comment:
Is creating a new causality region what we want here or do we want to insist that it is set explicitly and therefore have no default on the column?

Reply:
This is indeed subtle. First I proposed this rename because there is a chance that we will go for a deterministic implementation of file data sources that does not assume a total ordering, and therefore relies more on the concept of causality regions.

On explicit vs default, if we relied on this value being deterministic, then I'd say we should always set it explicitly and handle the causality region id generation ourselves. However when there is a tree of causality regions, which can grow at any leaf, this id would need to be a position in a tree rather than a sequential number.

So for now we have this simple id which should not be assumed to be deterministic, so it's fine for it to be generated by the DB.